testcases/data/lspci.txt

```diff
@@ -0,0 +1,22 @@
+00:00.0 "0604" "1014" "03dc" "" ""
```

Comment:
Maybe we should move all of the site specific test input into a top level directory?

This way an organisation could have a git repository of machines.xml and the other files needed (such as lspci output).

(this could come in another patch though)